trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/translator/Req2BoogieTranslator.java

```diff
@@ -97,6 +99,7 @@ public class Req2BoogieTranslator {
 
 	public static final String PROCEDURE_NAME = "myProcedure";
 	private static final String DOUBLE_ZERO = Double.toString(0.0);
+	private static final Boolean SUPPORT_HISTORY_VARS = false;

```

Comment:
* Why `Boolean` and not `bool`?
* Is it not somewhat risky to just have a hardcoded string here? Could the cases for which we need history vars be auto-detected? 
* At least add a comment that says what history vars are, and for what they are needed, and why this is a compile time switch.